src/openrct2/actions/PeepPickupAction.cpp

```diff
@@ -23,6 +23,15 @@ PeepPickupAction::PeepPickupAction(PeepPickupType type, EntityId spriteId, const
 {
 }
 
+
+void PeepPickupAction::AcceptParameters(GameActionParameterVisitor& visitor)
+{
+    visitor.Visit("type", _type);
+    visitor.Visit("entityId", _spriteId);
```

Comment:
There is a bit of inconsistentcy across the various staff/guest/peep actions for this field. They should all use the same name for this (id seems to be the most common)